src/interfaces/IProtocolGovernance.sol

```diff
@@ -10,11 +10,13 @@ interface IProtocolGovernance is IDefaultAccessControl, IERC165 {
     /// @param liquidationPremium Share of the MUSD value of assets of a vault, due to be awarded to a liquidator after a liquidation (multiplied by DENOMINATOR)
     /// @param maxDebtPerVault Max possible debt for one vault (nominated in MUSD weis)
     /// @param minSingleNftCapital Min possible MUSD NFT value allowed to deposit (nominated in MUSD weis)
+    /// @param maxNftsPerVault Max possible amount of NFTs for one vault
     struct ProtocolParams {
         uint256 liquidationFeeD;
         uint256 liquidationPremiumD;
         uint256 maxDebtPerVault;
         uint256 minSingleNftCollateral;
+        uint8 maxNftsPerVault;
```

Comment:
Struct layout could be optimised

https://fravoll.github.io/solidity-patterns/tight_variable_packing.html
https://docs.soliditylang.org/en/v0.8.17/internals/layout_in_storage.html